mne/epochs.py

```diff
@@ -506,6 +507,111 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
                            scalings=scalings, title_str=title_str,
                            show=show, block=block)
 
+    def plot_psds(self, fmin=0, fmax=np.inf, proj=False, n_fft=2048,
+                  picks=None, ax=None, color='black', area_mode='std',
+                  area_alpha=0.33, window_size=256, n_overlap=128,
+                  n_jobs=1, verbose=None):
+        """Plot the power spectral density across epochs
+
+        Parameters
+        ----------
+        fmin : float
+            Start frequency to consider.
+        fmax : float
+            End frequency to consider.
+        proj : bool
+            Apply projection.
+        n_fft : int
+            Number of points to use in Welch FFT calculations.
+        picks : array-like of int | None
+            List of channels to use.
+        ax : instance of matplotlib Axes | None
+            Axes to plot into. If None, axes will be created.
+        color : str | tuple
+            A matplotlib-compatible color to use.
+        area_mode : str | None
+            Mode for plotting area. If 'std', the mean +/- 1 STD (across
+            channels) will be plotted. If 'range', the min and max (across
+            channels) will be plotted. Bad channels will be excluded from
+            these calculations. If None, no area will be plotted.
+        area_alpha : float
+            Alpha for the area.
+        window_size : int, optional
+            Length of each window. The default value is 256.
+        n_overlap : int
+            The number of points of overlap between blocks. The default value
+            is 128 (window_size // 2).
+        n_jobs : int
+            Number of jobs to run in parallel.
+        verbose : bool, str, int, or None
+            If not None, override default verbose level (see mne.verbose).
+        """
+        return plot_epochs_psd(self, fmin=fmin, fmax=fmax, proj=proj,
+                               n_fft=n_fft, picks=picks, ax=ax,
+                               color=color, area_mode=area_mode,
+                               area_alpha=area_alpha, window_size=window_size,
+                               n_overlap=n_overlap, n_jobs=n_jobs,
+                               verbose=None)
+
+    def plot_psds_topomap(self, bands=None, vmin=None, vmax=None, proj=False,
+                          n_fft=2048, picks=None, window_size=256,
+                          n_overlap=128, layout=None, cmap='RdBu_r',
+                          agg_fun=np.sum, n_jobs=1, verbose=None):
```

Comment:
Also we need to approach the API a bit to the one we have in the AverageTFR object.. Second we should provie a 'normalize' flag (devide each frequency by sum of all frequencies).